src/main/kotlin/uk/gov/dluhc/printapi/database/entity/AnonymousElectorDocument.kt

```diff
@@ -41,40 +41,40 @@ class AnonymousElectorDocument(
 
     @field:NotNull
     @field:Size(max = 20)
-    var certificateNumber: String? = null,
+    var certificateNumber: String,
 
     @field:NotNull
     @Enumerated(EnumType.STRING)
-    var sourceType: SourceType? = null,
+    var sourceType: SourceType,
 
     @field:NotNull
     @field:Size(max = 255)
-    var sourceReference: String? = null,
+    var sourceReference: String,
 
     @field:Size(max = 255)
-    var applicationReference: String? = null,
+    var applicationReference: String?,
 
     @field:NotNull
     @Enumerated(EnumType.STRING)
-    var certificateLanguage: CertificateLanguage? = null,
+    var certificateLanguage: CertificateLanguage?,
 
     @Enumerated(EnumType.STRING)
-    var supportingInformationFormat: SupportingInformationFormat? = null,
+    var supportingInformationFormat: SupportingInformationFormat?,
```

Comment:
@Neil-Massey, I see that the supporting information format is in the `PrintRequest` for VACs, so assumed it might be different for different print requests. Is it not the case for the `AED`s as well? Can we not have 2 different print requests for AEDs in the future with 2 different supporting information formats? 
I saw there is only one supporting information format for AED, but can that change in the future?

Reply:
Will need to double check with the BAs - they only described reprint flows for LSD and electoral roll number re-indexing.  Will address in next PR if any changes required.